src/number_theory/cyclotomic/basic.lean

```diff
@@ -433,6 +459,54 @@ instance is_cyclotomic_extension [ne_zero ((n : ℕ) : A)] :
     { exact subalgebra.mul_mem _ hy hz },
   end }
 
+instance [ne_zero ((n : ℕ) : A)] :
```

Comment:
Why do you need to write `((n : ℕ) : A)`? Is `(n : A)` not enough?

Reply:
It indeed doesn't work, but I am not sure why. @ericrbg what do you think?

Reply:
Well, the reason is surely that the class `ne_zero` is written for `n : ℕ`, and the double coercion from `ℕ+` to `ℕ` to `R` probably confuses Lean. I don't know how we can avoid it.

Reply:
Yeah, I ran into this while writing it. I don't know how to deal with coe_trans; and I feel like any instance that does may be circular